pkg/storage/storage.go

```diff
@@ -0,0 +1,32 @@
+package storage
+
+import (
+	"errors"
+	"github.com/spf13/viper"
+	er "github.com/trustwallet/blockatlas/pkg/errors"
+	"github.com/trustwallet/blockatlas/pkg/logger"
+	"github.com/trustwallet/blockatlas/pkg/storage/redis"
+	"github.com/trustwallet/blockatlas/pkg/storage/sql"
+)
+
+var (
+	ErrNotFound     = errors.New("storage: obj not found")
+	ErrNotStored    = errors.New("storage: not stored")
+	ErrNotUpdated   = errors.New("storage: not updated")
+	ErrNotDeleted   = errors.New("storage: not deleted")
+	ErrAlreadyExist = errors.New("storage: object already exist")
+)
+
+var Redis redis.Redis
+var Postgres sql.PgSql
+
+func InitDatabases() {
+	err := Redis.Init(viper.GetString("observer.redis"))
+	if err != nil {
+		logger.Fatal(er.E(err), "Cannot connect to Redis")
+	}
```

Comment:
can we potentially use just one database? because if one of the databases is down - blockatlas will fail

Reply:
I like Redis to storage the block numbers, is quickly and don't need a lot of processing
But we can keep only one database to don't make the infra more complex